Components/Configs/IWebTemplateSettings.cs

```diff
@@ -10,7 +10,9 @@ public interface IWebTemplateSettings
         /// Set by application programmatically or in the Web.Config
         /// Only available in the Application Template
         /// </summary>
+#pragma warning disable CA1056
```

Comment:
Because this `#pragma` is peperred in a lot of places, just wondering if instead this warning could be suppressed in the GlobalSuppression file instead?

Reply:
Yup, made the change.

Reply:
Made changes to GlobalSuppressions